newIDE/app/public/external/yarn/yarn-main.js

```diff
@@ -0,0 +1,75 @@
+import { createPathEditorHeader, fileExists } from '../utils/path-editor.js';
+
+const electron = require('electron');
+const electronWindow = electron.remote.getCurrentWindow();
+const ipcRenderer = electron.ipcRenderer;
+const fs = require('fs');
+const remote = electron.remote;
+
+let yarn = null;
+
+const saveAndClose = pathEditor => {
```

Comment:
Let's rename this variable pathEditor to pathEditorHeader, to be consistent with the naming below (`const pathEditorHeader = createPathEditorHeader`)